### ISSUE TITLE
Ability to set a special closure on a park in DUP

### ISSUE THREAD

**Issue by JLWade** [bcgov/parks-reso-public#326]

**Comment by jpeersoxd** [comment]
Tested on TEST. No issues found. Moving to PO Review.

**Comment by JLWade** [comment]
@Christopher-walsh22 can we make one update?
- when passes are set to not required and the special closure toggle is on, can the line on public read only: "Limited Access" like it does when passes are required
  - unless I missed it and this new language is something that was determined elsewhere? @ndelraye 

![Image](https://github.com/bcgov/parks-reso-public/assets/38668130/67e96e3b-091d-4e76-8969-dc6bb532c0a5)

**Comment by Christopher-walsh22** [comment]
@JLWade Yeah I can that! I dont think you missed any conversations about that wording. It was the original template I had, must not have been changed.

**Comment by Christopher-walsh22** [comment]
This has been updated with text change request.

**Comment by JLWade** [comment]
meant to move this one to done yesterday!
